site/package.json

```diff
@@ -49,6 +50,8 @@
 		"@mui/system": "5.16.7",
 		"@mui/utils": "5.16.6",
 		"@mui/x-tree-view": "7.18.0",
+		"@radix-ui/react-icons": "1.3.0",
```

Comment:
So both were added when doing the shadcn installation, lucide is used by the default style and radix icons is used by the new-york style. Since we are going to use neither style, I need to update this for the icon library we are using going forward.

Reply:
Since it looks like Christin is using Lucide icons in Figma, I am going to update this to use those.